### PR TITLE
Make footer data

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?
Moves the footer links from hardcoded HTML into data to loop over. 

### UK

image

### US

image

### Australia

image

### International

image

## What is the value of this and can you measure success?
We can then share this with Dotcom-Rendering, as well as making it easier to edit on Frontend.

### CODE REVIEW

File: common/app/navigation/FooterLinks.scala [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  url: String,
+  dataLinkName: String,
+  extraClasses: String = ""
+)
+
+object FooterLinks {
+
+  // Footer column one
+
+  val complaintsAndCorrections = FooterLink("Complaints &amp; corrections", "/info/complaints-and-corrections", "complaints")
```
Comment (philmcmahon) [inline common/app/navigation/FooterLinks.scala]: does the amplisand still need to be escaped with this implementation?

**Comment by shtukas** [comment]
+1 From me.

**Review by nicl — COMMENTED** [review]
Could we just send one collection of footer links and do the editionalising in Frontend? I.e. rather than sending data for all editions and then filtering in DCR, just send e.g.:

    footerLinks: [ [ .. ], [ .. ], [ .. ] ]

or if we want to preserve the explicit columns as property names then:

    footerLinksColOne
    footerLinksColTwo
    ..

for example?

It might help simplify the data model, and reduce transferred data and burden on client to build up the entire model?

**Comment by gtrufitt** [comment]
> Could we just send one collection of footer links and do the editionalising in Frontend? 

Yes, this is a good idea!

**Review by nicl — APPROVED** [review]
Go go go!

**Pull request merged**